cfgov/unprocessed/apps/analytics-gtm/js/retirement-before-you-claim-listeners.js

```diff
@@ -120,8 +118,7 @@ import { closest } from '@cfpb/cfpb-atomic-component/src/utilities/dom-traverse.
     for (let i = 0, len = lifestyleBtns.length; i < len; i++) {
       lifestyleBtns[i].addEventListener('click', function (event) {
         const target = event.currentTarget;
-        // TODO: migrate off closest requirement, if possible.
-        const $container = closest(target, '.lifestyle-question_container');
+        const $container = target.closest('.lifestyle-question_container');
```

Comment:
A minor FYI on this one. The analytics-gtm scripts have corresponding GTM tags in tag manager, which are loaded by the google tag manager script in the head of the base template. These tags then make a call to load their corresponding script via our global jsl script loader. This loader will not load the script if `fetch` is not supported. However, the set of browsers that support `closest` is slightly smaller than the set that supports `fetch`. So, in theory there will be browsers with early `fetch` support that will error on this analytics script. It might be worth reaching out to the analytics team and asking if this specific tag (named similar to "retirement before you claim listeners") can be retired, in which case we'd delete this entire file.